### ISSUE TITLE
Add warning about print's buffer to docs

### ISSUE THREAD

**Issue by Alexorz** [apiaryio/dredd-hooks-python#16]
Hello,

I got a trouble, the hook function didn't be executed on my Mac when I was running `dredd`, and I found the TCP hook events handler had never been triggered.

Env:
* node.js v0.12.4
* dredd v1.0.5
* Python v3.4.3
* dredd-hooks v0.0.3

Tell me if you need more information.

**Comment by honzajavorek** [comment]
@Alexorz I'm keen to help you to debug this. Could you please share your `dredd.yml` and your hooks file?

**Comment by honzajavorek** [comment]
I tried to reproduce this, several combinations of versions, but so far unsuccessfully. @Alexorz may I ask you for more input so we could nail this down, please?

**Comment by honzajavorek** [comment]
@Alexorz We've just released new Dredd version (1.0.6). Could you please verify the problem persists?

**Comment by Alexorz** [comment]
Hello @honzajavorek , sorry for replying so late. I tried dredd v1.0.8, but it seems nothing changed.

Code of hooks.py:
````
import dredd_hooks

print('hook on')

@dredd_hooks.after_all
def my_after_all_hook(transactions):
  print('after all')
````

The log:
![qq20160317-2 2x](https://cloud.githubusercontent.com/assets/660581/13839635/7a368b48-ec57-11e5-8d08-b3f083aaca3c.png)

The `after_all` callback in `hooks.py` did not be triggered.

**Comment by w-vi** [comment]
@Alexorz because in python the stdout is buffered you have to flush it otherwise Dredd might not pick it up as it is stuck in a buffer ...  try ` sys.stdout.flush()` after the `print`

**Comment by Alexorz** [comment]
@w-vi , it works. So the hook is already working, just printed nothing...
Thanks.

**Comment by honzajavorek** [comment]
@Alexorz @w-vi Thanks! I'll leave this open, because we should mention this in docs.

**Comment by w-vi** [comment]
It has been fixed in #20 by wrapping all hook functions so that the output is flushed.
